__tests__/data/singular.json

```diff
@@ -478,117 +478,6 @@
       }
     }
   },
-  {
-    "description": "[Error]: (Android) Required field referring application missing for Session event",
```

Comment:
Why do we need to remove this test case?

Reply:
It was error case since this install_source was mandatory required earlier, so for missing it

Reply:
But we can keep this case and rather than error we can have payload with `""` in it

Reply:
Okay fixing this case.